Documentation/CrossPlatform/LeapMotionMRTK.md

```diff
@@ -0,0 +1,117 @@
+    - Select the MixedRealityToolkit game object in the hierarchy and select **Copy and Customize** to clone the default mixed reality profile.
+    
+    ![LeapMotionProfileClone](../Images/CrossPlatform/LeapMotion/LeapProfileClone.png)
+
+    - Select **Clone** in the input system profile to enable modification.
+
+    ![LeapMotionInputProfileClone](../Images/CrossPlatform/LeapMotion/LeapCloneInputSystemProfile.png)
+
+    - Open the **Input Data Providers** section, select **Add Data Provider** at the top, a new data provider will be added at the end of the list.  Open the new data provider and set the **Type** to **Microsoft.MixedReality.Toolkit.LeapMotion.Input > LeapMotionDeviceManager**
+
+    ![LeapAddDataProvider](../Images/CrossPlatform/LeapMotion/LeapAddDataProvider.png)
+
+    - The Leap Motion Data Provider contains the `LeapControllerOrientation` property which is the location of the leap motion controller. `LeapControllerOrientation.Headset` indicates the controller is mounted on a headset. `LeapControllerOrientation.Desk` indicates the controller is placed flat on desk. The default value is set to `LeapControllerOrientation.Headset`.  If the orientation is the desk, an extra property `LeapControllerOffset` will appear.  `LeapControllerOffset` is the anchor for the position of the desk leap hands.  The offset is calculated relative to the main camera position and the default value is (0,-0.2, 0.2) to make sure the hands appear in front and in view of the camera.
+
+    
+    `LeapControllerOrientation`: Headset (Default) |  `LeapControllerOrientation`: Desk 
+    :-------------------------:|:-------------------------:
+    ![LeapHeadsetGif](../Images/CrossPlatform/LeapMotion/LeapHeadsetOrientationExampleMetacarpals.gif)  |  ![LeapDeskGif](../Images/CrossPlatform/LeapMotion/LeapDeskOrientationExampleMetacarpals.gif)
+    ![LeapHeadsetGif](../Images/CrossPlatform/LeapMotion/LeapDevManagerHeadset.png) |     ![LeapHeadsetGif](../Images/CrossPlatform/LeapMotion/LeapDevManagerDesk.png)
+
+1. Testing the Leap Motion Data Provider
+    - After Leap Motion Data Provider has been added to the input system profile, press play, move your hand in front of the Leap Motion Controller and you should see the joint representation of the hand.
+
+1. Building your project 
+    - Navigate to **File > Build Settings**
+    - Only Standalone builds are supported if using the Leap Motion Data Provider.
+
+
+## Removing Leap Motion from the Project
+
+1. Close Unity
+1. Close Visual Studio, if it's open
+1. Open File Explorer and navigate to the root of the MRTK unity project
+    - Delete the **UnityProjectName/Library** directory
+    - Delete the **UnityProjectName/Assets/LeapMotion** directory
+    - Delete the **UnityProjectName/Assets/LeapMotion.meta** file
+- Reopen Unity
+
+If errors are logged after reopening, restart unity again.
+
+Debugging
+
+1. Leap Namespace errors missing
```

Comment:
Also I *think* that this section wasn't fully fleshed out (it seems like this was an outline for content yet to come)

Reply:
Yes that is an outline

Reply:
The debugging is going to have the three scenarios 
1. Importing Leap if the MRTK source is cloned from the repo
2. Importing Leap with the MRTK unity package into 2018.4
3. Importing Leap with the MRTK unity package into 2019.3